### PR TITLE
feat: support all Tier 2 KMP targets

### PR DESCRIPTION
**Pull request by rbraeunlich**
Refers to #628

### CODE REVIEW

**Comment by rbraeunlich** [comment]
Hi @charleskorn,
could you review the PR please?

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -19,9 +19,9 @@ jobs:
           - os: windows-latest
             tasks: mingwX64Test
           - os: macos-13
-            tasks: macosX64Test iosX64Test
+            tasks: macosX64Test iosX64Test tvosX64Test watchosX64Test
           - os: macos-latest
-            tasks: macosArm64Test iosSimulatorArm64Test
+            tasks: macosArm64Test iosSimulatorArm64Test tvosX64Test watchosSimulatorArm64Test
```
Comment (charleskorn) [inline .github/workflows/build.yml:24]: Should this be this?

```suggestion
            tasks: macosArm64Test iosSimulatorArm64Test tvosSimulatorArm64Test watchosSimulatorArm64Test
```

**Comment by charleskorn** [comment]
Thanks @rbraeunlich!

**Comment by rbraeunlich** [comment]
Is there a chance for you releasing a new version anytime soon @charleskorn?

**Pull request merged**